src/Microsoft.AspNetCore.BuildTools.ApiCheck/Program.cs

```diff
@@ -186,66 +201,120 @@ public static int Main(string[] args)
             {
-                if (compactOutput)
-                {
-                    Console.WriteLine($"ERROR: Exclusion is no longer necessary: Old type '{exclusion.OldTypeId}' member " +
-                        $"'{exclusion.OldMemberId}, New type '{exclusion.NewTypeId}' member '{exclusion.NewMemberId}'");
-                }
-                else
-                {
-                    Console.WriteLine($@"ERROR: The following exclusion is in the exclusion file, but is no longer necessary:
-    Old type: {exclusion.OldTypeId}
-    Old member: {exclusion.OldMemberId}
-    New type: {exclusion.NewTypeId}
-    New member: {exclusion.NewMemberId}");
+                Console.WriteLine();
+                Console.WriteLine();
+                Console.WriteLine("The following interfaces have one or more members added to them.");
 
+                foreach (var memberGrouping in newMembersOnInterfaces)
+                {
+                    Console.WriteLine(indent + memberGrouping.Key);
+                    Console.WriteLine(string.Join(Environment.NewLine,
+                        memberGrouping.Select(b => indent + indent + b.MemberId).ToList()));
                     Console.WriteLine();
                 }
             }
 
-            if (differences.Count > 0 || remainingExclusions.Count > 0)
+            breakingChangesToPrint.AddRange(newMembersOnInterfaces.SelectMany(t => t.ToList()));
+
+            foreach (var exclusion in incorrectBreakingChanges)
             {
-                if (!compactOutput && differences.Count > 0)
-                {
-                    Console.WriteLine("The process for breaking changes is described in: https://github.com/aspnet/Home/wiki/Engineering-guidelines#breaking-changes");
-                    Console.WriteLine("For how to add an exclusion to Api-check go to: https://github.com/aspnet/BuildTools/wiki/Api-Check#apicheck-exceptions");
-                }
+                Console.WriteLine(
+                    "ERROR: The following exclusion is in the exclusion file, but is no longer necessary:");
+                Console.WriteLine(JsonConvert.SerializeObject(exclusion, Formatting.Indented));
+                Console.WriteLine();
+            }
+
+            if (breakingChangesToPrint.Any())
+            {
+                Console.WriteLine();
+                Console.WriteLine();
+                Console.WriteLine("Following is the list of exclusions that either need to be added to the list of breaking changes, or the breaking changes themselves need to be reverted:");
+                Console.WriteLine(
+                    JsonConvert.SerializeObject(
+                        breakingChangesToPrint,
+                        Formatting.Indented,
+                        new JsonSerializerSettings
+                        {
+                            NullValueHandling = NullValueHandling.Ignore
+                        }));
+                Console.WriteLine();
```

Comment:
@smitpatel why write anything but this block and the bit about no-longer-necessary exclusions? Everything else written to the console in this method is redundant.